controllers/upgrade_handlers.go

```diff
@@ -259,45 +215,135 @@ func (r *ImageBasedUpgradeReconciler) prePivot(ctx context.Context, ibu *lcav1al
 	return ctrl.Result{}, nil
 }
 
-// startBackup manages backup flow and returns with possible requeue
-func (r *ImageBasedUpgradeReconciler) startBackup(ctx context.Context, backupCRs []*velerov1.Backup) (ctrl.Result, error) {
-	// sort backupCRs by wave-apply
-	sortedBackupGroups, err := backuprestore.SortByApplyWaveBackupCrs(backupCRs)
+func (r *ImageBasedUpgradeReconciler) postPivot(ctx context.Context, ibu *lcav1alpha1.ImageBasedUpgrade) (ctrl.Result, error) {
+	r.Log.Info("Checking cluster health")
+	// TODO: cluster health check
+
+	// Restore extra manifests
+	r.Log.Info("Applying DU extra manifests")
+	err := r.ExtraManifest.ApplyExtraManifests(ctx, r.Executor)
 	if err != nil {
+		if extramanifest.IsEMFailedError(err) {
+			upgradeFailedStatus(ibu, err.Error())
+			return doNotRequeue(), nil
+		}
 		return requeueWithError(err)
 	}
 
+	// Recover OADP configuration
+	r.Log.Info("Recovering OADP configuration")
+	err = r.BackupRestore.RestoreOadpConfigurations(ctx)
+	if err != nil {
+		if backuprestore.IsBRStorageBackendUnavailableError(err) {
+			upgradeFailedStatus(ibu, err.Error())
+			return doNotRequeue(), nil
+		}
+		return requeueWithError(err)
+	}
+
+	// Handle restore by OADP
```

Comment:
nit: I think comment and the log below it say one thing. Maybe the comment is not necessary.